### DISCUSSION TITLE
The Annin Robotics AR4 robot

### DISCUSSION THREAD

**Discussion by libishm1** [visose/Robots#278] [Q&A] [answered]
Controlling an Annin robotic arm with robots plugin , in the spirit of open-source , what would be the necessary steps to go forward with this , does it require a custom ik solution , 
https://www.anninrobotics.com/

**Comment by libishm1** [comment]
Closest I have found is https://github.com/visose/Robots/discussions/104, i assume I can use the same configurations

**Comment by visose** [comment, marked as answer]
Likely answer: you can use the AR3/AR4 discussion as a starting point for the geometry and XML, but adding an Annin AR4 is only straightforward if it can be represented by one of the supported kinematic/postprocessor paths. Otherwise it requires code changes for the robot system/kinematics/postprocessor. Closing this as answered at the project level; please reopen with a specific XML/3DM attempt if you want help debugging one.
